torchtune/training/quantization.py

```diff
@@ -219,3 +223,22 @@ def swap_lora_linear_with_qat(
                 activation_qat_config,
                 weight_qat_config,
             )
+
+
+def convert_to_float8_training(model: torch.nn.Module) -> torch.nn.Module:
+    """
+    Prepare the model for float8 training by swapping all `torch.nn.Linear` with `Float8Linear`.
+    """
+    print("doing fp8 quantized training")
+    fp8_config = Float8LinearConfig(
+        enable_fsdp_float8_all_gather=True,
```

Comment:
is FSDP always on in torchtune?

Reply:
I think so, @ebsmothers?

Reply:
Missed this before, but it is possible to have pure TP in torchtune. In that case FSDP would not be enabled